src/video.cpp

```diff
@@ -617,6 +696,7 @@ static std::vector<encoder_t> encoders {
   nvenc,
 #endif
 #ifdef _WIN32
+  quicksync,
```

Comment:
Should this go below the amd encoder? Otherwise I assume we would encode with quicksync if someone had an Intel igpu + amd gpu

Reply:
I don't think it actually matters today, since the Desktop Duplication API only supports instantiation on GPUs where the display is physically connected. `DuplicateOutput1()` would fail when called on the dGPU, since the display is connected to the iGPU and it would end up selecting QuickSync anyway.

In the future, we could add support for cross-adapter encoding, but that entails 4 copies per frame (iGPU texture -> iGPU staging texture -> dGPU staging texture -> dGPU texture) so might not have good performance anyway.